pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/PatchRoute.java

```diff
@@ -39,12 +43,15 @@ public Object processRequest(Request request, Response response, String userId)
             throw new RuntimeException("ColumnNameMap is null!");
         }
         String userIdRequest = UserUtil.getUserId(request);
-        if (UserUtil.checkUserAccess(null, userId, DBConstants.MR_VIEW, userIdRequest)
-                || UserUtil.checkUserAccess(null, userId, DBConstants.MR_ABSTRACTER, userIdRequest)
-                || UserUtil.checkUserAccess(null, userId, DBConstants.PT_LIST_VIEW, userIdRequest)) {
+        String requestBody = request.body();
+        Patch patch = GSON.fromJson(requestBody, Patch.class);
+        String realm = patch.getRealm();
```

Comment:
While you're modifying this, do you want to check the patch to make sure the userId in the patch is the same as the ID from line 45?

Reply:
Ah....I need to keep reading.